.travis.yml

```diff
@@ -13,12 +13,18 @@ matrix:
       env: TESTS=blocks FLOATX=float32
     - python: 2.7
       env: TESTS=blocks FLOATX=float32 DB=sqlite
+    - python: 2.7
+      env: TESTS=blocks FLOATX=float32 DB=mimir
     - python: 3.4
       env: TESTS=blocks FLOATX=float64
     - python: 2.7
+      env: TESTS=blocks FLOATX=float32 DB=mimir
```

Comment:
I am not sure that it is worth to run all these additional tests every time. We can at least skip Python 2.7 tests for Blocks and only do them for Blocks-examples.